lib/core/search-text/search-text-input.component.ts

```diff
@@ -98,6 +106,14 @@ export class SearchTextInputComponent implements OnInit, OnDestroy {
     @Output()
     reset: EventEmitter<boolean> = new EventEmitter();
 
+    /** Emitted when the search visibility changes. True when the search is active, false when it is inactive */
+    @Output()
+    searchVisibility: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+    /** Emitted when the clear button is clicked */
+    @Output()
+    clearButtonClicked: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
I don't think is a good practice to expose action buttons but we should expose behaviors of the component... for example, The search has been cleared or the search has been reset...